### ISSUE TITLE
Working regex marked as error in editor.  Subsequent closing brackets not recognized as matched.

### ISSUE THREAD

**Issue by NathanSmutz** [oakes/Nightcode#225]
The following code runs fine (ran it in a pre-parinfer version of Nightcode); but causes the current Nightcode editor to call out an error (same behavior in 2.1.4 and 2.3.5) :
```
(defn req-pct [s] (if-let [num-string (re-find #"(?<=Requirements\s{0,5})[-]?\d+(?=%)" s)]
                    (Double/parseDouble num-string)))
```

The editor seems to detect a (false?) error on the regular expression; then mark every close-paren/close-bracket/etc after the regular expression as an "Unmatched delimiter". 

![image](https://user-images.githubusercontent.com/2293340/27939522-43342c3c-627a-11e7-9c3b-1b36fd36b3ab.png)

It looks like it doesn't like one of:  `(?<=` or `(?=`.  
I'm seeing this with the jar versions of 2.1.4 and 2.3.5 on Windows 7 64-bit.

**Comment by oakes** [comment]
Fixed in master, thanks!

**Comment by oakes** [comment]
Just released 2.3.6 which fixes this issue.
